CONTRIBUTING.md

```diff
@@ -38,57 +38,46 @@ Finally, set the upstream remote to the official Spyder repo with:
 $ git remote add upstream https://github.com/spyder-ide/spyder.git
 ```
 
+
 ### Creating a conda environment or virtualenv
 
-If you use Anaconda you can create a conda environment with the following commands:
+If you use Anaconda you can create a conda environment and install the necessary dependencies as follows.
 
 ```bash
-$ conda create -n spyder-dev python=3
+$ conda create -n spyder-dev -c conda-forge --file requirements/conda.txt
 $ conda activate spyder-dev
+(spyder-dev) $ python install_dev_repos.py --editable
 ```
 
-You can also use `virtualenv` on Linux, but `conda` is **strongly** recommended:
-
-```bash
-$ mkvirtualenv spyder-dev
-$ workon spyder-dev
-```
-
-
-### Installing dependencies
-
-After you have created your development environment, you need to install Spyder's necessary dependencies. The easiest way to do so (with Anaconda) is
+If you are running on macOS, you will also need to install `python.app`, so the first line above should read
 
 ```bash
-$ conda install -c conda-forge --file requirements/conda.txt
+$ conda create -n spyder-dev -c conda-forge --file requirements/conda.txt python.app
 ```
 
-This installs all Spyder's dependencies into the environment.
-If you are running on macOS, you will also need to install `python.app`.
+You can also use `virtualenv` on Linux, but `conda` is **strongly** recommended:
 
 ```bash
-$ conda install python.app
+$ mkvirtualenv spyder-dev
+$ workon spyder-dev
+(spyder-dev) $ install -e .
+(spyder-dev) $ python install_dev_repos.py --editable --no-install spyder
 ```
 
-If using `pip` and `virtualenv` (not recommended), you need to `cd` to the directory where your git clone is stored and run:
-
-```bash
-$ pip install -e .
-```
 
 ### Running Spyder
 
 To start Spyder directly from your clone, i.e. without installing it into your environment, you need to run (from the directory you cloned it to e.g. `spyder`):
```

Comment:
```suggestion
To run Spyder from your clone in its development mode, with extra checks and options (pass `--help` to see them), launch it via the `bootstrap.py` script in the repo root directory:
```

Update this description to reflect that Spyder actually is installed, and briefly describe the point of `bootstrap,py` (over just running `spyder`)